### ISSUE TITLE
Wagtail 1.4 issue

### ISSUE THREAD

**Issue by SalahAdDin** [infoportugal/wagtail-modeltranslation#71]
I'm testing Wagtail 1.4 and i have this bug with model translatio:
```bash
Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x7f695eab8d90>
Traceback (most recent call last):
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/site-packages/django/core/management/commands/runserver.py", line 109, in inner_run
    autoreload.raise_last_exception()
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/site-packages/django/utils/autoreload.py", line 249, in raise_last_exception
    six.reraise(*_exception)
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 662, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/anadolu/core/models.py", line 20, in <module>
    from wagtail_modeltranslation.models import TranslationMixin
  File "/run/media/salahaddin/Datos/Proyectos/Trabajo/adkm/lib/python3.5/site-packages/wagtail_modeltranslation/models.py", line 16, in <module>
    from wagtail.wagtailadmin.views.pages import get_page_edit_handler,\
ImportError: cannot import name 'get_page_edit_handler'

```

This bug makes Wagtail Model translation useless, we couldn't use when official release will be release.

**Comment by Alexandre27** [comment]
We're aware of this error and there is already an issue (issue #67) to make wagtail-modeltranslation compatible with Wagtail 1.4. @hmartiniano made a pull request that solves this issue but since we're going to remove the obligation to inherit from TranslationMixin and SnippetMixin and make this classes deprecated we didn't merge that pull request.

**Comment by SalahAdDin** [comment]
Ah, ok.

**Comment by Alexandre27** [comment]
@SalahAdDin If you want to test Wagtail 1.4rc1 use the version available on branch release/v0.5 as it is already compatible with Wagtail 1.4.

**Comment by SalahAdDin** [comment]
Well, for now i don't need it, so, i'll wait for the new release version, and for an documentation update.
